pulpcore/pulpcore/app/models/publication.py

```diff
@@ -14,12 +15,18 @@ class Publication(Model):
             create this Publication.
     """
 
+    complete = models.BooleanField(default=False)
     created = models.DateTimeField(auto_now_add=True)
 
     publisher = models.ForeignKey('Publisher', on_delete=models.CASCADE)
 
     repository_version = models.ForeignKey('RepositoryVersion', on_delete=models.CASCADE)
 
+    class Meta:
+        indexes = [
```

Comment:
I think models.BooleanField(default=False, db_index=True) will do the same thing.

Reply:
Agreed with @asmacdo and would be more consistent with the other models.